### DISCUSSION TITLE
Support for already existing zip file?

### DISCUSSION THREAD

**Discussion by herndlm** [Elendev/nexus-composer-push#60] [General]
I wonder if I can specify a zip file I already have to be used instead?
Although I love the default features we have some use cases where I want to push after some custom build ran that created the zip file.

If this is not working yet - would you consider accepting a contribution that adds this?

**Comment by Elendev** [comment]
Hello @herndlm, it's a good idea for a new feature.
I would love to accept such a contribution, feel free to propose a pull request, and I'll have a look :-)

Cheers
